src/openrct2/drawing/TTFSDLPort.cpp

```diff
@@ -1,6 +1,6 @@
 // Adapted from freetype.h in order to avoid C-style casts.
 
-#define FT_LOAD_TARGET_ALT(x) (static_cast<FT_Int32>((x)&15) << 16)
+#define FT_LOAD_TARGET_ALT(x) (static_cast<FT_Int32>((x) & 15) << 16)
```

Comment:
You can turn off clang-tidy for this file, it’s all third-party code in order to avoid a dependency of SDL2-TTF. (We should probably move it to the appropriate folder.)